### DISCUSSION TITLE
QR Control examples

### DISCUSSION THREAD

**Discussion by RobbyBrts** [gopro/labs#1114] [Q&A]
Hello,

I am new to this. Is there a website or other medium where I can find QR codes that I can use to scan with my GoPro 10 Black?

**Comment by lyricsdude** [comment]
https://gopro.github.io/labs/control/extensions/

**Reply by RobbyBrts** [reply]
I don't see any QR codes

**Reply by lyricsdude** [reply]
you fill a four letter code in the bottom generator and a value. then it generates a qr-code that you can scan. You can do it 1 by 1 and click "make metadata permanent". Then your gopro remembers each scanned qr-code.

**Reply by RobbyBrts** [reply]
Okey, but.... the question was;
Is there a website or other medium where I can find QR codes that I can use to scan with my GoPro 10 Black?

**Reply by lyricsdude** [reply]
those QR codes you can scan once you generated them...

**Reply by RobbyBrts** [reply]
Okey, thanks.

Anyone else have another suggestion/idea?

**Reply by lyricsdude** [reply]
haha did you even check the whole page: https://gopro.github.io/labs/control/extensions/
you don't see the QR code at the bottom?
